ecosystem/sep-0036.md

```diff
@@ -0,0 +1,137 @@
+## Preamble
+
+```
+SEP: 0036
+Title: Regulated Assets Transacting
+Author: Howard Chen <@howardtw>
+Status: Draft
+Created: 2020-10-26
+Version 0.1.0
+```
+
+## Simple Summary
+
+Regulated Assets provide ecosystem support for assets that require an issuer’s approval (or a delegated third party’s approval, such as a licensed securities exchange) on a per-transaction basis. It standardizes the identification of such assets as well as defines the protocol for performing compliance checks and requesting issuer approval.
+
+This SEP is based on [SEP-0008](sep-0008.md), but the issuer no long has to a co-signer on the account holding the regulated asset. It also updates the API of the approval server to better align with other active SEPs.
+
+**Target Audience**: Asset issuers, issuance platforms, wallet developers, and exchanges
+
+## Motivation
+
+Stellar aims to be an ideal platform for issuing securities. As such, it must provide tools to comply with various regulatory requirements. Regulation often requires asset issuers to monitor and approve every transaction involving the issued asset and to enforce an assortment of constraints. This SEP provides support for these requirements.
+
+## Overview
+
+Implementing a Regulated Asset consists of these parts:
+- [Stellar.toml]: Issuers will advertise the existence of an Approval Service and the approval criteria via their stellar.toml file.
+- [Approval Server]: HTTP protocol for transaction signing.
+- [Transaction Composition]: Wallets will have to put operations that fully authorize the accounts for the asset, transact the asset, and deauthorize the accounts in a single transaction.
```

Comment:
I think this is the place where we disambiguate between the operations the client actually wants and the operations that are necessary to allow it to happen. The client wants an operation to be executed and it needs to be preceded and followed up with operations that authorize/deauthorize the transaction.

 Also, its not clear if the Wallet/Client or Approval Server is the one to add the auth/deauth operations to the set of transaction that actually do what the client wants.

Reply:
> Also, its not clear if the Wallet/Client or Approval Server is the one to add the auth/deauth operations to the set of transaction that actually do what the client wants.

How is it not clear which party is the one building the transaction? I thought this sentence states that `Wallets will have to put operations that...`. Got suggestions?

How about this?
```
Wallets will have to build and sign a transaction containing operations that fully authorize the accounts for the asset, transact the asset, and deauthorize the accounts.
```
By mentioning the **sign**, it should be clear that no party should modify the transaction.